pipeline_dp/accumulator.py

```diff
@@ -366,3 +368,10 @@ def add_accumulator(self,
     def compute_metrics(self) -> float:
         return dp_computations.compute_dp_sum(self._sum,
                                               self._params.mean_var_params)
+
+    def clamp_value(self, value):
```

Comment:
Nit: use [np.clip](https://numpy.org/doc/stable/reference/generated/numpy.clip.html). It should work for the list as well